content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md

```diff
@@ -29,12 +29,12 @@ a simple nginx web server to demonstrate the concept.
 
 ## Example flow with endpoint termination
 
-The following is the example of the flow described in the
+The following is the example flow described in the 
 [Termination of Pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)
 document.
 
-Let's say you have a Deployment containing of a single `nginx` replica
-(just for demonstration purposes) and a Service:
+Let's say you have a Deployment containing a single `nginx` replica
+(say just for the sake of demonstration purposes) and a Service:
```

Comment:
The previous statement was better suited as new one does not add any new changes.

Reply:
Agreed